### ISSUE TITLE
Operator "in" not working as expected with "chunk" of text

### ISSUE THREAD

**Issue by DarkBee** [twigphp/Twig#3037]
Consider following snippet

```
{% set foobar_chunk %}
foobar
{% endset %}

{% set foobar = 'foobar'  %}

{{ 'foo' in foobar_chunk ? 1 : 0 }} {# output : 0 expected 1 #}
{{ 'foo' in foobar ? 1 : 0 }} {# output: 1 expected 1 #}
```

The fact the first test fails is that `foobar_chunk` gets converted into a `Twig_Markup` and the [`twig_in_filter`](https://github.com/twigphp/Twig/blob/2.x/src/Extension/CoreExtension.php#L923) does not account for it in the source code.

Is the intentional or can we just test if `$compare` is an instance of `Twig_Markup` and convert it into a string and then do the test?

**Comment by fabpot** [comment]
This should work as of Twig 1.40/2.9. Are you using an older version?

**Comment by DarkBee** [comment]
Tested this on 2.10.0 on [twigfiddle](https://twigfiddle.com/dwh8vx)

**Comment by fabpot** [comment]
fixed in #3049
